CIP-0136/README.md

```diff
@@ -0,0 +1,210 @@
+Implementors: []
+Discussions:
+    - https://github.com/cardano-foundation/CIPs/pull/878
+Created: 2024-07-17
+License: CC-BY-4.0
+---
+
+## Abstract
+
+The Conway ledger era ushers in on-chain governance for Cardano via [CIP-1694 | A First Step Towards On-Chain Decentralized Governance](https://github.com/cardano-foundation/CIPs/blob/master/CIP-1694/README.md), with the addition of many new on-chain governance artifacts.
+Some of these artifacts support the linking of off-chain metadata, as a way to provide context to on-chain actions.
+
+The [CIP-100 | Governance Metadata](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0100) standard provides a base framework for how all off-chain governance metadata can be formed and handled.
+This standard was intentionally limited in scope, so that it can be expanded upon by more specific subsequent CIPs.
+
+This proposal aims to provide a specification for the off-chain metadata vocabulary that can be used to give context to Constitutional Committee (CC) votes.
+
+## Motivation: why is this CIP necessary?
+
+The high-level motivation for this proposal is to provide a standard which improves legitimacy of Cardano's governance system.
+
+### Clarity for governance action authors
+
+Governance action authors are likely to have dedicated a significant amount of time to making their action meaningful and effective (as well as locking a significant deposit).
+If this action is not able to be ratified by the CC, it is fair for the author to expect a reasonable explanation from the CC.
+
+Without reasonable context being provided by the CC votes, authors may struggle to iterate upon their actions, until they are deemed constitutional.
+This situation could decrease perceived legitimacy in Cardano's governance.
+
+### Context for other voting bodies
+
+By producing a standard we hope to encourage all CC members to attach rich contextual metadata to their votes.
+This context should show CC member's decision making is fair and reasonable.
+
+This context allows the other voting bodies to adequately check the power of the CC.
+
+### CC votes are different to other types of vote
+
+The CC and their votes are fundamentally very different from the other voting bodies.
+This makes reusing standards from these voting bodies problematic.
+
+### Inclusion within interim constitution
+
+Cardano's [Interim Constitution Article VI Section 4](https://github.com/IntersectMBO/interim-constitution/blob/75155526ce850118898bd5eacf460f5d68ceb083/cardano-constitution-0.txt#L330) states:
+
+```txt
+Constitutional Committee processes shall be transparent.
+The Constitutional Committee shall publish each decision.
+When voting no on a proposal, the Committee shall set forth the basis
+for its decision with reference to specific Articles of this Constitution
+that are in conflict with a given proposal.
+```
+
+This mandates that the CC must provide rationale for at least `no` votes.
```

Comment:
```suggestion
This mandates that the CC must provide rationale for `Yes, `No` and `Abstain` votes.
```
Some might not realize the true meaning behind the machine-readable "no" in this context due to the way it is written, and helps preventing any confusion or unnecessary effort to explain their decision when abstaining from voting.

Kudos to the authors of this CIP! I truly appreciate the meticulous effort they put into it. Everything else looks great to me. 
Awesome work @Ryun1 , @Willburn 😃👍

Reply:
I am neutral on if it should be manditatory for all governance actions or only no as it depends on interpretation (I personally think same as Hornan but maybe others do not). Lowest common denominator is manditatory for  a no vote. 

I am also curious if we should up the summary field to 300 chars? Have anyone tried to write a summary with the rationales so far? 200 seems a bit too limiting its not a CC tweet :)

Reply:
> ```suggestion
> This mandates that the CC must provide rationale for `Yes, `No` and `Abstain` votes.
> ```
> Some might not realize the true meaning behind the machine-readable "no" in this context due to the way it is written, and it would help preventing any confusion or unnecessary effort to explain their decision when abstaining from voting.
> 
> Kudos to the authors of this CIP! I truly appreciate the meticulous effort they put into it. Everything else looks great to me. 
> Awesome work @Ryun1 , @Willburn 😃👍

This change suggests that the constitution mandates a rationale for all decision types, which isn't accurate. While we could recommend providing a rationale for all decisions, suggesting it is mandated would be misleading.

Reply:
Here’s a corrected version of your message while maintaining its essence:

"This is an interesting point. I didn't see any mention of CIP-136 in the Constitution, nor any requirement to use a metadata standard except for governance actions in Article 3, Section 6.

Article 6, Section 5 states: 'The Constitutional Committee shall publish each decision.' This could leave a lot of room for interpretation regarding whether this requirement applies only to a 'No' vote or to all votes. However, there’s still no mention of mandatory metadata standards for the vote rationales.

Don't get me wrong; I'm just trying to clarify potential confusion in good faith. If this isn't optimal and you have a better suggestion, I would be glad to hear it."

Reply:
> This is an interesting point. I didn't see any mention of CIP-136 in the Constitution, nor any requirement to use a metadata standard except for governance actions in Article 3, Section 6.
> 
> Article 6, Section 5 states: 'The Constitutional Committee shall publish each decision.' This could leave a lot of room for interpretation regarding whether this requirement applies only to a 'No' vote or to all vote types. However, there’s still no mention of mandatory metadata standards for the vote rationales.
> 
> Don't get me wrong; I'm just trying to clarify potential confusion in good faith. If this isn't optimal and you have a better suggestion, I would be glad to read it.

Could you please confirm which version of the constitution you are referring to? Article VI Section 5 in the Interim Constitution (https://constitution.gov.tools/en/interim-constitution) refers to tooling. Section 4 refers to publishing a rationale, but it says when "voting no on a proposal".

If the language in this CIP references the constitution, it should accurately represent the current constitution. Maybe it would be safer not to reference the constitution in this CIP, given it can change.

Reply:
Article VI Section 4 is what I was referencing

https://constitution.gov.tools/en/interim-constitution#section-3-4

Reply:
Thank you, Ryan. Yes, as Phil mentioned, it seems I don’t have the correct version. I apologize for that. In mine, it’s in Section 6 😅. I’ll definitely rely on the one from the Intersect repo moving forward. But I’m glad it’s mentioned in Section 4.

Perhaps using the word 'mandate' isn’t the best approach, or as Phil mentioned, referencing the constitution. These are metadata we’re going to use anyway, and over the months, they’ll become standard practice. I wouldn’t mind if there’s no formal mandate to use them, because I’m sure we’ll adopt them as standard practice regardless.

My original intention was to prevent people from thinking that 'mandatory' includes 'abstain from voting' based on how the sentence originally sounded. But if we can easily find a way to phrase it that avoids this confusion, my the issue will be resolved. 😇👍

Reply:
Please see latest related discussions on this topic via https://github.com/cardano-foundation/CIPs/pull/878#discussion_r1899087912